packages/twenty-front/src/pages/settings/SettingsWorkspaceMembers.tsx

```diff
@@ -27,7 +49,17 @@ const StyledButtonContainer = styled.div`
   margin-left: ${({ theme }) => theme.spacing(3)};
 `;
 
+const StyledTable = styled(Table)`
+  margin-top: ${({ theme }) => theme.spacing(0.5)};
+`;
+
+const StyledTableHeaderRow = styled(Table)`
+  margin-bottom: ${({ theme }) => theme.spacing(1.5)};
```

Comment:
it's a bit a code smell that you are using .5 units
Usually if you have that is that the design meant that the height was important and not the margin